projects/packages/waf/src/brute-force-protection/class-shared-functions.php

```diff
@@ -36,12 +46,12 @@ public static function format_whitelist() {
 		}
 		if ( is_multisite() && current_user_can( 'manage_network' ) ) {
 			$formatted['global'] = array();
-			$global_whitelist    = self::get_global_whitelist();
-			if ( false === $global_whitelist ) {
-				// If the global whitelist has never been set, check for a legacy option set prior to 3.6.
-				$global_whitelist = get_site_option( 'jetpack_protect_whitelist', array() );
+			$global_allow_list   = self::get_global_allow_list();
+			if ( false === $global_allow_list ) {
+				// If the global allow list has never been set, check for a legacy option set prior to 3.6.
+				$global_allow_list = get_site_option( 'jetpack_protect_whitelist', array() );
```

Comment:
Maybe we should change this option name as well? We could have some sort of migration method to replicate the old option into the new one perhaps?

Reply:
I'm thinking we address the network/site options in a separate effort/release. Initially, I didn't want to mess with these site options to avoid extra complexity. What do you think?

Calypso references the "jetpack_protect_global_whitelist" site option directly by name, so it will require some extra effort to manage compatibility between everyone's Jetpacks and Calypso when creating that option migration. It will also be the second time this option has been re-named: first "jetpack_protect_whitelist", then "jetpack_protect_global_whitelist", and now likely "jetpack_protect_global_allow_list". The existing legacy option checks can likely be cleaned up as well when we address this.

Either way, I think the migration should be in a separate PR to keep this one nice and simple.

Reply:
Agreed!